apps/backend/src/models/proxy_config.py

```
@@ -0,0 +1,264 @@
+            raise ValueError("file_path must point to a .conf file")
+        
+        return value.strip()
+    
+    def __repr__(self):
+        return f"<ProxyConfig(id={self.id}, service='{self.service_name}', subdomain='{self.subdomain}')>"
+
+
+class ProxyConfigChange(Base):
+    """
+    Time-series table for tracking changes to proxy configurations
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)